### ISSUE TITLE
How can I change the sequence of phases, the order attribute does not change that?

### ISSUE THREAD

**Issue by marcuspecht** [WELTEN/dojo-ibl#162]

**Comment by Titogelo** [comment]
For now you will be able to rename the phase. It is not the ideal solution but the ideal one requires more coding as it needs to modify all the activities under the phases.

Example of what implications has what you are suggesting:

1) initial setup
Phase 1 | Phase 2 | Phase 3

2) I want to drag phase 1 to the end
Phase 2 | Phase 3 | Phase 1

3) Each activity has a number that indicates the phase it belongs to. To re-organise phases, I also need to do the same with the phase number of the activities. If you want to do the change above... it means that all the numbers of the activities need to be modify. Activities in 'Phase 2' will have the number '1'. Activities in 'Phase 3' will have the number '2' and activities in 'Phase 1' will have the number '3'. 

So let's try first if renaming phases works. If it doesn't i will invest some time to implement a more advanced functionality.

**Comment by Titogelo** [comment]
we need to reorder the phases and all the activities should be updated as well. Following the example explained above.

**Comment by Titogelo** [comment]
I tried to used drag and drop to sort phases, but I've found usability several issues in Chrome. For instance: while dragging a phase the page scrolls automatically and I was not able to solve it. Thus, I went for an easier solution with arrows to move everything left or right.

<img width="671" alt="screen shot 2016-07-25 at 12 00 54" src="https://cloud.githubusercontent.com/assets/1598632/17097765/84b98016-525f-11e6-86ed-b177b1e5efa0.png">
